index.html

```diff
@@ -176,6 +177,17 @@ <h2>Board</h2>
 					<div slot="main">This is a board</div>
 				</ts-board>
 			</article>
+			<article>
+				<h2>Pager</h2>
+				<ts-pager id="mypager" pages="8"> </ts-pager>
+				<script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The length of this JS script (6) exceeds the maximum set to 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3ALongJavaScriptCheck)